packages/tutanota-crypto/lib/hashes/Sha1.js

```diff
@@ -1,20 +1,17 @@
 // @flow
 // $FlowIgnore[untyped-import]
-import sjcl from "./lib/sjcl"
-import {assertWorkerOrNode} from "../../common/Env"
-
-assertWorkerOrNode()
+import sjcl from "../internal/sjcl"
 
 const sha1 = new sjcl.hash.sha1()
 
-export const HASH_LENGTH = 32
+const SHA1_HASH_LENGTH_BYTES = 20
```

Comment:
This is not used or exported and also the value has been changed from 32 to 20. Should this line just be deleted?